RecoPixelVertexing/PixelTriplets/plugins/PixelQuadrupletGenerator.cc

```diff
@@ -117,7 +159,8 @@ void PixelQuadrupletGenerator::hitQuadruplets(const TrackingRegion& region, Orde
   std::vector<float> bc_r(4), bc_z(4), bc_errZ(4);
 
   // Loop over triplets
-  for(const auto& triplet: triplets) {
+  for(auto iTriplet = tripletsBegin; iTriplet != tripletsEnd; ++iTriplet) {
+    const auto& triplet = *iTriplet;
```

Comment:
Note to self: remove the change, it is just a leftover of some intermediate developments that got effectively removed in squash.